### ISSUE TITLE
Add extra threads to a session

### ISSUE THREAD

**Issue by llin60** [oneapi-src/oneVPL#103]
Hi, I've been using oneVPL to do some video processing tasks, and I found that in some cases when I connect a decode session with a vpp session, the `MFXVideoCORE_SyncOperation` function in the vpp session takes a long time to execute, leading to a slow overall processing speed.
Based on my understanding, this `MFXVideoCORE_SyncOperation` calls the internal scheduler to run the existing awaiting tasks, I'm guess the slow processing speed is probably caused by the limited thread number assigned to the scheduler in the session, so I wonder if there is any code I can refer to to manually add some extra threads to a session.
Thanks in advance.

**Comment by shepark** [comment]
Yes, there's `AsyncDepth`, https://spec.oneapi.io/versions/latest/search.html?q=AsyncDepth
Please review `sample_decode` tool under `tools/legacy/sample_decode` folder for the proper implementation for that case.
And command line option is `-async`.

**Comment by akwrobel** [comment]
@llin60 do you have any remaining questions post the comment above?
